src/TJNull_OSCP_Report_Template.md

```diff
@@ -0,0 +1,336 @@
+---
```

Comment:
What is the difference with this template from the already existing ones?

Reply:
not much, this was a slight iteration of the whoisflynn template to be more in line with OSCP official template, but in md. Details here: https://github.com/tjnull/TJ-JPT. Ultimately, little.